arcgis-ios-sdk-samples/Maps/Display device location/CustomContextSheet.swift

```diff
@@ -104,9 +104,9 @@ class CustomContextSheet: UIView {
         }
         
         //other buttons
-        for i in 0..<images.count {
+        for (index, image) in images.enumerated() {
```

Comment:
The following construct is preferred over `enumerated()`:

```swift
for index in collection.indices {
    let element = collection[index]
    …
}
```

Because the first element of the tuple is an *offset*, not an index. It always starts at `0`, which is not the case for the indices of every collection. The [documentation](https://developer.apple.com/documentation/swift/enumeratedsequence) uses the term `offset` throughout.

Reply:
Thanks for the tip! I didn't realize it didn't necessarily use the indexes.